### ISSUE TITLE
Runpod.io template for it so that anyone can use it no matter what computer they have

### ISSUE THREAD

**Issue by mp3pintyo** [NVlabs/Sana#35]
Unfortunately, due to its hardware requirements, many people cannot run it on their own computers. That's why I created a Runpod.io template for it so that anyone can use it no matter what computer they have:
[https://runpod.io/console/deploy?template=x39vw9jp5w&ref=2pdhmpu1](https://runpod.io/console/deploy?template=x39vw9jp5w&ref=2pdhmpu1)
(It is worth choosing A40 among the GPUs. It is only $0.39 per hour)
How to use it:
1. Register on the RunPod interface and top up some money to the virtual account (a few dollars).
2. Open the template link that I shared with you.
3. Selecting an A40 machine in Deploy
4. If you don't see it in the template, search for: Sana by Mp3Pintyo
5. After it starts, you have to wait until the Connect button turns purple. Let's click on it. After 15 minutes, you can use the 2 Connect to the HTTP button. (It downloads and installs a lot in the background. )
Port 15432: Starts a GUI to generate
Port 8888: starts the JupyterLab graphical interface where we access the files in the /content folder.
6. If you are no longer using the system, stop the Pod and delete it!

**Comment by lawrence-cj** [comment]
Good one. Thanks man.

**Comment by FurkanGozukara** [comment]
I also published a kaggle free account notebook - you can use for free 30 hours every week - working decent - 15 second per image

so you can generate 7200 images every week for free

https://medium.com/@furkangozukara/nvidia-labs-developed-sana-model-weights-and-gradio-demo-app-published-check-out-this-amazing-new-264943998f15

**Comment by AfterHAL** [comment]
> Good one. Thanks man.

"&ref=2pdhmpu1" in the URL --> Affiliation 
Template Referral Program with 1% Commission.

mp3pintyo should be the one who says "thank you".

**Comment by mp3pintyo** [comment]
> > Good one. Thanks man.
> 
> "&ref=2pdhmpu1" in the URL --> Affiliation Template Referral Program with 1% Commission.
> 
> mp3pintyo should be the one who says "thank you".

Of course, you can delete the ref link from the end. If you want, I can edit the original post and delete it.

**Comment by AfterHAL** [comment]
> > > Good one. Thanks man.
> > 
> > 
> > "&ref=2pdhmpu1" in the URL --> Affiliation Template Referral Program with 1% Commission.
> > mp3pintyo should be the one who says "thank you".
> 
> Of course, you can delete the ref link from the end. If you want, I can edit the original post and delete it.

I'm not telling people to remove it.
I just think it's more fair to let people know.
